appveyor.yml

```diff
@@ -1,9 +1,9 @@
-version: '{build}'
-skip_branch_with_pr: true
-shallow_clone: true
-build_script:
-  - ps: ./tools/appveyor/build.ps1
-  - ps: "$deployPath = $env:APPVEYOR_BUILD_FOLDER + '/deploy'; bash ./tools/appveyor/deploy.sh $deployPath" 
-  - ps: "[System.IO.Compression.ZipFile]::CreateFromDirectory($env:APPVEYOR_BUILD_FOLDER + '/deploy', $env:APPVEYOR_BUILD_FOLDER + '/deploy.zip')"
-artifacts:
+version: '{build}'
+skip_branch_with_pr: true
+shallow_clone: true
+build_script:
+  - ps: ./tools/appveyor/build.ps1
+  - ps: "$deployPath = $env:APPVEYOR_BUILD_FOLDER + '/deploy'; bash ./tools/appveyor/deploy.sh $deployPath" 
+  - ps: "[System.IO.Compression.ZipFile]::CreateFromDirectory($env:APPVEYOR_BUILD_FOLDER + '/deploy', $env:APPVEYOR_BUILD_FOLDER + '/deploy.zip')"
+artifacts:
```

Comment:
@AffectedArc07 

What did you do, and why is this showing up in a merge from upstream commit.